test/nextjournal/clerk/builder_test.clj

```diff
@@ -37,10 +38,14 @@
     (is (= ["book.clj"] (builder/expand-paths {:paths ["book.clj"]}))))
 
   (testing "invalid args"
-    (is (thrown? Exception (builder/expand-paths {})))
-    (is (thrown? Exception (builder/expand-paths {:paths-fn 'foo})))
-    (is (thrown? Exception (builder/expand-paths {:paths-fn "hi"})))
-    (is (thrown? Exception (builder/expand-paths {:index ["book.clj"]})))))
+    (is (thrown? ExceptionInfo (builder/expand-paths {})))
+    (is (thrown? ExceptionInfo (builder/expand-paths {:paths-fn :foo})))
+    (is (thrown? ExceptionInfo (builder/expand-paths {:paths-fn 'foo})))
+    (is (thrown? ExceptionInfo (builder/expand-paths {:paths-fn 'foo/bar})))
+    (is (thrown? ExceptionInfo (builder/expand-paths {:paths-fn 'clojure.core/inc})))
+    (is (thrown? ExceptionInfo (builder/expand-paths {:paths-fn 'clojure.core/+})))
```

Comment:
Add a case for a non existing var in an existing namespace? e.g. `'clojure.core/this-var-does-not-exist`